ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -3430,6 +3431,18 @@ spanning_tree:
       priority: < priority >
 ```
 
+### System Boot Settings
+
+```yaml
+boot:
```

Comment:
As discussed offline, we should adjust this to:
```yaml
boot:
  # Set the Aboot password
  secret:
    hash_algorithm: < md5 | sha512 | default -> sha512 >
    key: "< hashed_password >"
```

Reply:
Sure, will do.